### ISSUE TITLE
Uncaught Error: Cannot find module '@toast-ui/editor/dist/i18n/ru-ru'

### ISSUE THREAD

**Issue by bel1k0v** [nhn/tui.editor#1684]
Module not found: Error: Package path ./dist/i18n/ru-ru is not exported from package /var/www/project_manager/node_modules/@toast-ui/editor (see exports field in /var/www/project_manager/node_modules/@toast-ui/editor/package.json)

## To Reproduce
1. Go to app.js
2. import '@toast-ui/editor/dist/i18n/ru-ru'
3. Save
4. npm run watch (npm mix watch)

## Expected behavior
Change editor language

## Desktop
 - Ubuntu 20.04
 - npm 7.19.1
 - laravel-mix 6.0.25
 - toast-ui/editor 3.0.1

**Comment by stfenjobs** [comment]
same here,

**toast-ui/editor 3.0.1**

```
Module not found: Package path ./dist/i18n/zh-cn is not exported from package /Users/xxxx/node_modules/@toast-ui/editor (see exports field in /Users/xxxx/node_modules/@toast-ui/editor/package.json)
  1 | import React from 'react';
  2 | import {Editor, EditorProps} from '@toast-ui/react-editor';
> 3 | import '@toast-ui/editor/dist/i18n/zh-cn';

```

**Comment by js87zz** [comment]
@bel1k0v 
The bug has been fixed in `v3.0.2`. Thanks!
